src/rpc/RPCHelpers.cpp

```diff
@@ -1350,6 +1350,19 @@ parseTaker(boost::json::value const& taker)
         return Status{RippledError::rpcBAD_ISSUER, "invalidTakerAccount"};
     return *takerID;
 }
+
+ripple::Issue
+parseIssue(boost::json::object const& issue)
+{
+    Json::Value jv;
+    if (issue.contains(JS(issuer)))
+        jv["issuer"] = issue.at(JS(issuer)).as_string().c_str();
```

Comment:
I don't see anything in validation that checks there are these fields and that they are strings.. in this case i believe if i send an object where the `issuer` is a number or anything but string it will crash same way here.. wdyt?

Reply:
This might be an old issue we had. I have add the check here.